src/c++/perf_analyzer/test_concurrency_manager.cc

```diff
@@ -149,7 +149,7 @@ class TestConcurrencyManager : public TestLoadManagerBase,
     if (use_mock_infer_) {
       return std::make_shared<MockConcurrencyWorker>(thread_config);
     } else {
-      return std::make_shared<TestConcurrencyWorker>(
+      return std::make_shared<ConcurrencyWorkerMockedInferInput>(
```

Comment:
I'm realizing now that this if/else is a little hard to understand the distinction just from reading these lines. I ask myself what's the difference between `MockConcurrencyWorker` and `ConcurrencyWorkerMockedInferInput`.

Reply:
I think this will get cleaned up when i pull the memory manager code out of this.
Right now, its a bit unclean because its all so coupled.

Reply:
What he said. I forgot to note it in my self review, but all that nastiness goes away after Elias's story.